src/scripts/main.js

```diff
@@ -1,3 +1,52 @@
 'use strict';
 
 // write code here
+const headline = document.querySelector('thead');
```

Comment:
It's a good practice to add a check if the element was found on the page. If the 'thead' element is not found, your script will throw an error in the following lines. You could add a condition to check if 'headline' is not null before adding an event listener.